petab/visualize/data_overview.py

```diff
@@ -55,13 +54,14 @@ def get_data_per_observable(measurement_df: pd.DataFrame) -> pd.DataFrame:
     Arguments:
         measurement_df: PEtab measurement data frame
     Returns:
-        data_per_observable:
-            Pivot table with number of data points per observable and condition
+        Pivot table with number of data points per observable and condition
     """
 
     my_measurements = measurement_df.copy()
-
-    my_measurements[PREEQUILIBRATION_CONDITION_ID].fillna('', inplace=True)
+    if PREEQUILIBRATION_CONDITION_ID not in my_measurements:
+        my_measurements[PREEQUILIBRATION_CONDITION_ID] = ''
+    else:
+        my_measurements[PREEQUILIBRATION_CONDITION_ID].fillna('', inplace=True)
```

Comment:
```suggestion
    index = [SIMULATION_CONDITION_ID]
    if PREEQUILIBRATION_CONDITION_ID in my_measurements:
        my_measurements[PREEQUILIBRATION_CONDITION_ID].fillna('', inplace=True)
        index.append(PREEQUILIBRATION_CONDITION_ID)
```
then `index=index` below, to avoid unspecified optional columns in the result?

Reply:
thx, yes